File: Proyectos/club_ciclista_ruby/el_club_ruby/lib/prueba_club.rb
#encoding: UTF-8


require_relative 'bicicleta.rb'
require_relative 'ciclista.rb'
require_relative 'club.rb'
  
module El_Club
  unClub = Club.new("los intrépidos", "trevenque 2, bajo B")
  miBici = Bicicleta.new
  unCiclista = Ciclista.new("Pepe",30,miBici,unClub)
  otraBici = Bicicleta.new(5, "negra")
  otroCiclista = Ciclista.new("Ana", 40, otraBici,unClub)
  unClub.nuevoSocio(unCiclista)
  unClub.nuevoSocio(otroCiclista)
  unClub.misSocios()
  
  
end

File: Proyectos/ClubCiclistaJava/ElClub/src/el_club/PruebaClub.java
// Ejemplo de uso de colecciones de objetos.

package el_club;
 
public class PruebaClub {

    /**
     * @param args the command line arguments
     */
    public static void main(String[] args) {
       
        Club unClub = new Club("los intrépidos", "trevenque 2, bajo B");
        Bicicleta miBici = new Bicicleta();
        Ciclista unCiclista = new Ciclista("Pepe",30,miBici,unClub);
        Bicicleta otraBici = new Bicicleta (5, "negra");
        Ciclista otroCiclista = new Ciclista("Ana", 40, otraBici,unClub);
        unClub.nuevoSocio(otroCiclista);
        unClub.nuevoSocio(unCiclista);
        unClub.misSocios();
        
    }
    
}
